.github/configs/labeler.yml

```diff
@@ -0,0 +1,17 @@
+version: 1
+labels:
+  - label: "xsmall-diff"
+    size-below: 10
+  - label: "small-diff"
+    size-above: 9
+    size-below: 100
+  - label: "medium-diff"
+    size-above: 99
+    size-below: 500
+  - label: "large-diff"
+    size-above: 499
+    size-below: 1000
+  - label: "xlarge-diff"
+    size-above: 999
+  - label: "testing-required-internal-e2e"
```

Comment:
Please change to the agreed upon `testing-needed-e2e`

Reply:
Actually, please change it to `testing-needed-e2e-fast`. We will figure out how/if to apply `testing-needed-e2e-full` to indicate when a full run is required vs the Kind-based workflow. For now we can make it so admins of this repo can manually apply `testing-needed-e2e-full` when/if it's deemed required.

Reply:
Updated in the latest patch as suggested.